docs/user/azure/credentials.md

```diff
@@ -46,14 +46,6 @@ You need to save the client secret values to configure your local machine to run
 
 You can get client secret for your service principal using the Azure [portal][sp-creds-portal] or the Azure [cli][sp-creds-cli]
 
-The default location of the service principal file is in ${HOME}/.azure/osServicePrincipal.json. An alternative location can be specified by setting the AZURE_AUTH_LOCATION environmental variable. You can generate a service principal file by running the following commands from a shell:
-
-````console
```

Comment:
@abhinavdahiya, if you need to [grant additional permissions to the service principal](https://github.com/openshift/openshift-docs/pull/16067/files#diff-60b65c7fe69b1bc3bc2dc7894d447bb1R124), how can you create it during the wizard?

Reply:
The user was already provided instructions to create a SP (step 1) https://github.com/openshift/installer/pull/2388/files#diff-b65bf060215e0e8accb9d785e95ee0fdR8

and then (step 2) https://github.com/openshift/installer/pull/2388/files#diff-b65bf060215e0e8accb9d785e95ee0fdR12 provides the instructions to get the required permissions.

And this point the SP credentials are ready to be used for installer (step 4) https://github.com/openshift/installer/pull/2388/files#diff-b65bf060215e0e8accb9d785e95ee0fdR43

This sections that is being removed, creates a NEW SP, which is not correct in the context of this doc.